agent/lib_aws_sdk_php.c

```diff
@@ -86,19 +86,21 @@
 
 //clang-format on
 */
-void nr_lib_aws_sdk_php_sqs_handle(nr_segment_t* segment,
+void nr_lib_aws_sdk_php_sqs_handle(nr_segment_t* auto_segment,
                                    char* command_name_string,
                                    size_t command_name_len,
                                    NR_EXECUTE_PROTO) {
   char* command_arg_value = NULL;
+  nr_segment_t* segment = NULL;
```

Comment:
The code readability would increase if this local variable would be named `message_segment`.